docfx/doxygen2syntax_test.cc

```diff
@@ -436,5 +461,32 @@ TEST(Doxygen2Syntax, Class) {
   EXPECT_EQ(actual, kExpected);
 }
 
+TEST(Doxygen2Syntax, Namespace) {
+  auto constexpr kExpected = R"yml(syntax:
+  contents: |
+    namespace google::cloud::mocks { ... };
+  source:
+    id: google::cloud::mocks
+    path: google/cloud/mocks/mock_stream_range.h
+    startLine: 30
+    remote:
+      repo: https://github.com/googleapis/google-cloud-cpp/
+      branch: main
+      path: google/cloud/mocks/mock_stream_range.h)yml";
+  pugi::xml_document doc;
+  doc.load_string(kNamespaceXml);
+  auto selected =
+      doc.select_node("//*[@id='namespacegoogle_1_1cloud_1_1mocks']");
+  ASSERT_TRUE(selected);
+  YamlContext ctx;
+  ctx.parent_id = "test-only-parent-id";
```

Comment:
nit: unnecessary?

Reply:
I was going to remove it, but I think I will leave it for consistency with (future) tests.